apps/saas_kernel/priv/protos/saastle.proto

```
@@ -0,0 +1,206 @@
+syntax = "proto3";
+
+package saastle.v1;
```

Buf (1.55.1)

Files with package "saastle.v1" must be within a directory "saastle/v1" relative to root but were in directory "apps/saas_kernel/priv/protos".

(PACKAGE_DIRECTORY_MATCH)